Update version

keckelt opened the pull request:
Close #19 

* Upgrade to JupyterLab v3
* Update+Cleanup npm dependencies
* Update dockerfile + add instructions to README
* Replace yarn with npm

keckelt committed:
Update version

dependabot[bot] committed:
Bump path-parse from 1.0.6 to 1.0.7 (#17)

Bumps [path-parse](https://github.com/jbgutierrez/path-parse) from 1.0.6 to 1.0.7.
- [Release notes](https://github.com/jbgutierrez/path-parse/releases)
- [Commits](https://github.com/jbgutierrez/path-parse/commits/v1.0.7)

---
updated-dependencies:
- dependency-name: path-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

keckelt committed:
Merge pull request #18 from jku-vds-lab/scalability

trrack update, differential state model and fixes/cleanup

keckelt committed:
Merge branch 'scalability' of github.com:JKU-ICG/jupyterlab_nbprovenance into scalability

keckelt committed:
Replace yarn with npm in README #20

keckelt committed:
Pin to juypterlab v2

keckelt committed:
remove yarn lock, update ignore files

keckelt committed:
remove travis config

keckelt committed:
update dependencies

keckelt committed:
Update dockerfile

keckelt committed:
Merge pull request #21 from jku-vds-lab/replace_yarn

keckelt committed:
Merge branch 'scalability' of github.com:JKU-ICG/jupyterlab_nbprovenance into scalability

keckelt committed:
Merge branch 'master' into jupyter_lab_v3

keckelt committed:
remove unnecessary/outdated files

keckelt committed:
Update license and repo URLs

keckelt committed:
Update Dockerfile

keckelt committed:
Update dependencies #19

ostie reviewed (changes requested):
There seems to be some problem with changing the activeCellIndex.
When a node with a different activeCell is selected, a new activeCellChanged node is added.
My guess is the event is now emitted asynchronously and the pauseTracking property no longer works.

ostie committed:
jupyterlab 3 update fix

- fix activeCell changing by "placeholder cell" rendering
- fix tracking issues
- fix cypress test
